v2/validator.go

```diff
@@ -72,6 +84,8 @@ func getErrorMessage(e validator.FieldError) string {
 		msg = fmt.Sprintf("%s field needs a uuid", fieldName)
 	case dtoNoneEmptyStringTag:
 		msg = fmt.Sprintf("%s field should not be empty string", fieldName)
+	case dtoRFC3986UnreservedCharTag:
+		msg = fmt.Sprintf("%s field only allows unreserved characters as defined in https://tools.ietf.org/html/rfc3986#section-2.3", fieldName)
```

Comment:
Put `ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_.~` in the message so user doesn't have to look it up?

Reply:
Updated